assets/src/application/ui/forms/modalForm/RenderModalForm.tsx

```diff
@@ -40,22 +42,33 @@ const RenderModalForm: React.FC<RenderModalFormProps> = ({
 	};
 
 	return (
-		<Modal
-			title={title}
-			visible={true}
-			onOk={onClose}
-			okText={__('Submit')}
-			okButtonProps={submitButton}
-			cancelText={__('Reset')}
-			cancelButtonProps={resetButton}
-			onCancel={onClose}
-			width={'80%'}
-			wrapClassName='ee-modal-form'
-			bodyStyle={{ padding: 0 }}
-		>
-			<div className='form-body'>{children}</div>
-		</Modal>
+		<div>
+			<EspressoModal
+				aria-label='Warning about next steps'
+				cancelButtonProps={resetButton}
+				cancelText={__('Reset')}
+				onClose={onClose}
+				okButtonProps={submitButton}
+				okText={__('Submit')}
+				title={title}
+			>
+				<div className='form-body'>{children}</div>
+			</EspressoModal>
+		</div>
```

Comment:
Why `aria-label='Warning about next steps'` for `RenderModalForm`?

Reply:
that remained from `reach-ui`,
now this PR is blocked until we will update our fork of `antd`